### PR TITLE
chore: update package.json

### PR DESCRIPTION
**Pull request by subhamg**

### CODE REVIEW

**Comment by coolsoftwaretyler** [comment]
Hey @subhamg - thanks for your PR!

We are working on a [milestone for getting the repository ready for more contribution](https://github.com/mobxjs/mobx-devtools/milestone/1) so we'll probably hold off on this until we finish up https://github.com/mobxjs/mobx-devtools/pull/129. If you want to help out, there are some details in that PR about the status of the change.

File: packages/mobx-devtools-mst/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "build": "cross-env NODE_ENV=production webpack"
   },
   "peerDependencies": {
-    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0",
+    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^6.13.0",
```
Comment (coolsoftwaretyler) [inline packages/mobx-devtools-mst/package.json:26]:
```suggestion
    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
```

I think ^6.0.0 should work to do what ^6.13.0 is doing here. Was there a specific reason you had wanted to include that?

**Comment by moharamfatema** [comment]
Any updates on this?

**Comment by andykog** [comment]
Closed in favour of #156

**Pull request closed without merging**